### ISSUE TITLE
gMCP archived on CRAN

### ISSUE THREAD

**Issue by rpodcast** [kornl/gMCP#15]
Hello, you may be aware of this already but gMCP has been archived on CRAN as of today (24FEB2018): https://cran.r-project.org/web/packages/gMCP/ .  Are there plans to fix any issues the CRAN maintainers identified?

**Comment by kornl** [comment]
Yes. I'll try to fix this together with the crossover package. The crossover package was already fixed, but CRAN rejected it, because their Windows build had (IMO) a broken rJava installation. I tried to notify them, but never got any response. Since gMCP will have the same problem, I was not that eager to fix it and instead wait till someone with more influence or persistence and a package with rJava dependence will have more success... Will try it again this week and see whether they have fixed it or answer me this time.
Thanks for opening the issue nevertheless!

**Comment by rpodcast** [comment]
@kornl Thank you for the quick response!  I recall a discussion I had with Dong Xi a few years ago about the possibility of separating the interactive Java interface to a separate package, enabling less of a dependency overhead for the actual graphical testing functions.  We use the backend functions of `gMCP` extensively (without the GUI) as part of some custom pipelines.  If you'd like to explore that possibility, I would be glad to open a separate issue to discuss those details.

**Comment by kornl** [comment]
The win-builder problem still persists. But I will now write to different places to get them fix it.

**Comment by xidongdxi** [comment]
It is great to reconnect with @kornl @rpodcast here. Many thanks for discussing the issue. Hope it could be solved and bring gMCP back to CRAN. In addition, I'd also be interested to discuss/contribute to further development of gMCP or its variant.

**Comment by kornl** [comment]
This took (too) long. But now it's back on CRAN.
